guac/certifier-scorecard.md

```diff
@@ -0,0 +1,233 @@
+---
+layout: page
+permalink: /guac/scorecard-certifier/
+redirect_from: /scorecard-certifier/
```

Comment:
```suggestion
```

No need to add a `redirect_from` to a new page. Existing pages had that because we put the GUAC docs under `/guac` after merging the Trustify community in.